Musico/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,38 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[ show edit update destroy ]
+
+  # GET /users/1 or /users/1.json
+  def show
+  end
+
+  # PATCH/PUT /users/1 or /users/1.json
+  def update
+    respond_to do |format|
```

Comment:
@mohseengs do we need respond_to here, as we are rendering only one type of format